CMakeLists.txt

```diff
@@ -11,12 +11,19 @@ endif()
 set(BUILD_CONFIG Debug)
 set(CMAKE_BUILD_TYPE ${BUILD_CONFIG})
 set(NAME scribbles)
+set(CLASSNAME "${NAME}ClassFiles")
 
 project(${NAME})
 
-add_executable(${NAME}
-	main.cpp
+add_executable(${NAME} main.cpp)
+
+add_library(
```

Comment:
This indicates that you chose the library approach, but you did everything else as if you chose the other. 😛

If that's a mistake, add `env` source files to `SOURCES` that's passed to `add_executable`.

Reply:
It is my bad 😅